.github/workflows/sql-extraction.yml

```diff
@@ -0,0 +1,42 @@
+name: SQL Extraction
+
+on:
+  push:
```

Comment:
If we don't filter on branches, it will be default run all branches right?

Reply:
Yup. Should I restrict this to master?

Reply:
I don't think this matters. Your call.